pyne/mesh.py

```diff
@@ -1300,6 +1307,9 @@ def structured_get_divisions(self, dim):
         """
         self._structured_check()
 
+        ## sometimes the dim is the ascii of the 'x', 'y', 'z'
+        #if isinstance(dim, int):
+        #    dim = chr(dim)
```

Comment:
If this is no longer the case, let's remove all these commented lines.

Reply:
My bad, forgot to clean up here. Will change it in next commit.